internal/services/firewall/firewall_policy_rule_collection_group_resource_test.go

```diff
@@ -794,6 +802,14 @@ resource "azurerm_firewall_policy_rule_collection_group" "test" {
       destination_urls      = ["www.google.com/en"]
       terminate_tls         = true
       web_categories        = ["News"]
+      http_headers {
+        name  = "head_foo"
+        value = "value_bar"
+      }
+      http_headers {
+        name  = "head_bar"
+        value = "value2"
+      }
```

Comment:
It looks like we're supposed to be testing the update path for this property but these values are the same as the ones provided in the `completePremium` config. Should these be different?

Reply:
Thanks, Updated.

```
--- PASS: TestAccFirewallPolicyRuleCollectionGroup_updatePremium (263.64s)
PASS
```